src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -87,11 +91,15 @@ const ReleaseContent = () => {
   const { publication } = release;
 
   const allMethodologies = useMemo<MethodologyLink[]>(() => {
-    const methodologies = publication.methodologies.map(methodology => ({
-      key: methodology.id,
-      title: methodology.title,
-      url: `/methodology/${methodology.id}/summary`,
-    }));
+    const methodologies = publication.methodologies
+      .filter(methodology => methodology.status === 'Approved')
+      .map(methodology => ({
+        key: methodology.id,
+        title: methodology.title,
+        url: isPreRelease
+          ? `/publication/${publication.id}/release/${release.id}/prerelease/methodologies/${methodology.id}`
+          : `/methodology/${methodology.id}/summary`,
+      }));
```

Comment:
If we only ever list methodologies which are in status Approved, I'm wondering why we didn't just filter them in the back end, in method `GetManageContentPageViewModel`?

It doesn't seem great to expose unapproved methodologies in the back end response especially to pre-release users who maybe shouldn't even be aware of them.

Reply:
I think you're right - looks like I've just had an epic brain fart.

I'll close this PR and reopen a new one.